examples/project/Earthfile

```diff
@@ -0,0 +1,7 @@
+VERSION 0.6
+PROJECT foo/bar
+
+FROM alpine:3.15
+
+hello:
+    RUN echo "Hello World"
```

Comment:
I think that this is not a strong enough example of how to use the `PROJECT` keyword. I wouldn't add this in, to keep our examples clean.

Reply:
SGTM. Perhaps I'll revisit for the new secret fetching and feature flag.